frontend/uddhar-frontend/src/public/pages/Error.jsx

```
@@ -0,0 +1,77 @@
+          {/* Error Message */}
+          <h1 className="text-4xl font-bold text-gray-800 mb-4">
+            Looks like you&apos;re lost
+          </h1>
+          <p className="text-gray-600 text-lg mb-8">
+            The page you are looking for is not available!
+          </p>
+  
+          {/* Error Code */}
+          <div className="flex items-center justify-center mb-8">
+            <span className="text-6xl font-bold text-red-500 mr-4">404</span>
+            <div className="h-16 w-px bg-gray-300"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)